app/src/main/java/com/dimelo/sampleapp/ConfigRc.java

```diff
@@ -0,0 +1,51 @@
+package com.dimelo.sampleapp;
+
+import android.content.Context;
+
+import com.dimelo.dimelosdk.main.Dimelo;
+import com.google.firebase.iid.FirebaseInstanceId;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+public class ConfigRc {
+    public static Dimelo setupDimelo(Context context) {
+        String secret = BuildConfig.DIMELO_SDK_SECRET; //edit in gradle.properties
+        String domainName = BuildConfig.DIMELO_SDK_DOMAIN_NAME; //edit in gradle.properties
+        Dimelo.setup(context);
+        Dimelo.interactiveNotification = true;
```

Comment:
Why add this? The "interactive" property is to be able to reply directly from the push notification, right? This is supposed to be handled by the notification payload we receive from the server.

Reply:
yes exactly , i can remove this